pipettes/firmware/hardware_config.c

```diff
@@ -140,7 +146,7 @@ static uint16_t get_spi_pins_lt(GPIO_TypeDef* for_handle) {
     }
 }
 
-static uint16_t get_motor_driver_pins_ht(GPIO_TypeDef* for_handle) {
+static uint16_t get_motor_driver_pins_ht(GPIO_TypeDef* for_handle, bool for_diag0) {
```

Comment:
I'm always uncomfortable with having a bool in a function argument that makes it return a completely different thing. Let's either have this return a quick struct by-value or have a second function that's like `get_diag0_pin_{ht,lt}`.